<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doubanLogin</class>
 <widget class="QWidget" name="doubanLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>doubanLogin</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QWidget#doubanLogin{
	background-color: rgb(255, 255, 255);
}*/</string>
  </property>
  <widget class="QLabel" name="loginTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>5</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>65</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="userAlias">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>55</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#userAlias{
	background-color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="text">
    <string>  Mail/UserName</string>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>105</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#password
{
	
	background-color:rgb(195, 195, 195);
}</string>
   </property>
   <property name="text">
    <string>  Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="verification">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>155</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#verification{
	
	background-color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="text">
    <string>  Verification</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aliasInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>55</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
     border: none;
     background:rgb(195, 195, 195);
 }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>105</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#passwdInput{
	border: none;
	background-color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="verificaitonInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>155</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#verificaitonInput{
	border: none;
	background-color: rgb(195, 195, 195);
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="captchaContainer" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>155</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#captchaContainer{
	
	background-color: rgb(195, 195, 195);
}</string>
   </property>
  </widget>
  <widget class="QToolButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>205</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#loginButton{
	border:none;
	background-color: rgb(157, 214, 197);
}
QToolButton#loginButton:hover{
	color:white;
	border:none;
}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QToolButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>205</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#cancleButton{
	border:none;
	background-color: rgb(157, 214, 197);
}
QToolButton#cancleButton:hover{
	border:none;
	color:white;

}
</string>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QLabel" name="errMsg">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>12</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#errMsg{
	color:red
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
